src/views/pages/Login/__tests__/container.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import configureStore from 'redux-mock-store'
+
+import LoginContainer from '../container'
+
+describe('LoginContainer', () => {
+  const mockStore = configureStore([])()
+
+  it('matches snapshot', () => {
+    const container = shallow(
+      <LoginContainer
+        store={mockStore}
+        onSubmit={jest.fn}
+      />
+    )
+      .dive()
+      .dive()
```

Comment:
can we use our `diveTo` utility here?

Reply:
I know about this util, but I dont want to use special export for tests. I want sets dive manually